pkg/upgrade/control_plane.go

```diff
@@ -419,6 +426,7 @@ func (u *ControlPlaneUpgrader) updateBootstrapConfig(name string, machine *clust
 		}
 	}
 	bootstrap.Spec.JoinConfiguration.NodeRegistration = nodeRegistration
+	bootstrap.Spec.JoinConfiguration.Discovery.BootstrapToken = &kubeadmv1beta1.BootstrapTokenDiscovery{}
```

Comment:
Is there an existing one to copy over?

Reply:
the problem with copying it over is that it's probably expired. Clearing it like this will generate a new one that won't be expired

Reply:
oh cool